playlist_length/main.py

```diff
@@ -11,8 +11,8 @@
 from huepy import bold, green, red
 from tqdm import tqdm
 
-from .utils import pluralize
-from .__version__ import __version__
+from playlist_length.utils import pluralize
```

Comment:
Change to relative imports

Reply:
If I do that it can't be used as a standalone file